functions-and-scope.js

```diff
@@ -53,13 +87,26 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade([6, 4, 5]) geeft xxxx
 // averageGrade([8, 9, 4, 6, 10]) geeft xxxx
 
+let averageGrades = 0;
+
+function arrayOfGrades() {
+    let sum = 0;
+    for (let i = 0; i < grades2.length; i++) {
+        sum += grades2[i] // zie boven let uit 2a.
+        averageGrades = sum / grades2.length
```

Comment:
Ik zou dit na de for-loop doen, omdat het resultaat hetzelfde is, maar in theorie nu onnodige CPU power vraagt